Lib/ufo2ft/outlineCompiler.py

```diff
@@ -73,12 +73,19 @@ def compile(self):
         """
         self.otf = TTFont(sfntVersion=self.sfntVersion)
 
-        self.vertical = False
-        for glyph in self.allGlyphs.values():
-            if (hasattr(glyph, "verticalOrigin") and
-                    glyph.verticalOrigin is not None):
-                self.vertical = True
-                break
+        # only compile vertical metrics tables if vhea metrics a defined
+        vertical_metrics = [
+            "openTypeVheaVertTypoAscender",
+            "openTypeVheaVertTypoDescender",
+            "openTypeVheaVertTypoLineGap",
+            "openTypeVheaCaretSlopeRise",
+            "openTypeVheaCaretSlopeRun",
+            "openTypeVheaCaretOffset",
```

Comment:
@moyogo that _all_ vhea metrics be defined in fontinfo.plist seems a bit too much. Maybe only check if `VertTypo*` are present and leave the `VheaCaret*` stuff as optional?

Reply:
OK, I’ll change the fallback values to get a horizontal caret in
https://github.com/googlei18n/ufo2ft/blob/5f2e3514be3f625e3e40bbefdc8a3bbe59a0420b/Lib/ufo2ft/fontInfoData.py#L360-L368